sw/cheri/common/console.hh

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright lowRISC contributors.
+ * Licensed under the Apache License, Version 2.0, see LICENSE for details.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#pragma once
+
+#include "fmt.hh"
+#include "sonata-devices.hh"
+#include "../../common/defs.h"
+#include <platform-uart.hh>
+
+#include "console.hh"
+
+#define CC_BOLD "1"
+#define CC_RED "31"
+#define CC_GREEN "32"
+#define CC_RESET "0"
```

Comment:
I think I'll probably still want them to have the full escape code and them just write them directly.

Reply:
I agree, but I want this PR to only translate the old way of logging to print. I'm happy to tackle that in a follow-up PR.